i18n/locales/de-DE/account.json

```
@@ -35,18 +35,76 @@
             "successMessage": "Eine Bestätigungs-E-Mail mit einem Link zum Zurücksetzen des Passworts wurde versandt, wenn die angegebene E-Mail-Adresse registriert ist."
         },
         "customer": {
-            "accountType": "Accounttyp",
-            "salutation": "Anrede",
-            "firstName": "Vorname",
-            "lastName": "Nachname",
+            "accountType": {
+                "label": "Accounttyp",
+                "errorRequired": "Bitte wählen Sie einen Accounttyp aus",
+                "business": "Business",
+                "private": "Privat"
+            },
+            "salutation": {
+                "label": "Anrede",
+                "errorRequired": "Bitte Anrede auswählen",
```

```suggestion
                               "errorRequired": "Bitte wählen Sie eine Anrede aus",
```

for matching error messeages

---

Done